docs/core/integration-quality-scale/rules/iqs047.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "IQS047 - The documentation describes the supported functionality, including entities, and platforms"
+---
+
+IQS047 is as follows:
+
+The documentation describes the supported functionality, including entities, and platforms
+
+## Reasoning
+
+Users should be able to understand what value the integration will add for their (to be bought) device.
+This will help them create better expectations.
+
+For example, if a user is looking for a new fridge, we should try to be clear about what they can expect from the integration.
+If the integration only supports checking if the door is open or closed, they would be disappointed if they expected to be able to view the temperature of the fridge.
+
+## Example implementation
+
+```markdown
+## Supported functionality
+
+The integration will provide a sensor for all the measurements and will provide a switch to turn the device on and off.
+```
```

Comment:
This is a bad example. Could not really think of something better

Reply:
Let me try and find one...